instrumentation/factory_bot/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
From what I see, this repo is our fork of the `opentelemetry-ruby-contrib`. Is there a clear plan to merge some of these back upstream? (thinking if we really need the license etc.)

Reply:
Yes, this is a 100% generic solution that could be accepted by the upstream repo so I will propose that.